comtypes/GUID.py

```diff
@@ -54,7 +54,7 @@ def __unicode__(self):
 
     def __cmp__(self, other):
         if isinstance(other, GUID):
-            return cmp(binary(self), binary(other))
+            return (binary(self), binary(other))
```

Comment:
It looks strange to me because `__cmp__` method should return "a negative value if self < other , positive if self > other , and zero if they were equal" (c). So why does it return a 2-tuple now???